index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <div id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="original trombones logo">
+      <div>
+        <div><a class="nav-link" href="#features">Features</a></div>
+        <div><a class="nav-link" href="#how-it-works">How It Works</a></div>
+        <div><a class="nav-link" href="#pricing">Pricing</a></div>
+      </div>
+    </nav>
+  </div>
```

Comment:
This code should be changed to be header instead of div tag

Reply:
Try to explain "why" for your required change here too.